saleor/graphql/shipping/mutations/shippings.py

```diff
@@ -171,6 +171,9 @@ def success_response(cls, instance):
 class ShippingZipCodeRulesCreateInputRange(graphene.InputObjectType):
     start = graphene.String(required=True, description="Start range of the zip code.")
     end = graphene.String(required=False, description="End range of the zip code.")
+    inclusion_type = ZipCodeRuleInclusionTypeEnum(
+        required=False, description="Inclusion type for given zip code rules."
+    )
```

Comment:
I've added it as non required field to not break existing Saleor Dashboard code. I can change it.